lib/modules/datasource/nuget/index.spec.ts

```diff
@@ -302,6 +309,160 @@ describe('modules/datasource/nuget/index', () => {
       );
     });
 
+    describe('determine source URL from nupkg', () => {
+      beforeEach(() => {
+        GlobalConfig.set({
```

Comment:
I think we should do a reset in before each for all tests to not cause side effects later

Reply:
Done in e7bc6f53c5ac9410d1f5229d560f93211fffe959